packages/big-design/src/components/ButtonGroup/styled.tsx

```diff
@@ -0,0 +1,52 @@
+import { theme as defaultTheme } from '@bigcommerce/big-design-theme';
+import styled, { css } from 'styled-components';
+
+import { ButtonProps } from '../Button';
+import { StyleableButton } from '../Button/private';
+import { FlexItem } from '../Flex';
+
+interface StyledFlexItemProps {
+  isVisible: boolean;
+}
+
+interface StyledButtonProps extends ButtonProps {
+  isFirst?: boolean;
+  isLast?: boolean;
+}
+
+export const StyledFlexItem = styled(FlexItem)<StyledFlexItemProps>`
+  ${(props) =>
+    !props.isVisible &&
+    css`
+      position: absolute;
+      visibility: hidden;
+      z-index: ${({ theme }) => -theme.zIndex.tooltip};
+    `}
+`;
```

Comment:
We tend to destruct the props.

```suggestion
  ${({ isVisible }) =>
    !isVisible &&
    css`
      position: absolute;
      visibility: hidden;
      z-index: ${({ theme }) => -theme.zIndex.tooltip};
    `}
`;
```